src/file-manager/fm-directory-view.c

```diff
@@ -5056,6 +5083,7 @@ add_extension_menu_items (FMDirectoryView *view,
 				       gtk_action_get_name (action),
```

Comment:
I honestly think this is not a big deal, let's not worry about it for now, otherwise too many changes might introduce unintended issues

Reply:
Should i change the code like this?
```
		if (add_action) {
			G_GNUC_BEGIN_IGNORE_DEPRECATIONS
			action = gtk_action_new (action_name,
						 escaped_label,
						 NULL,
						 NULL);

			gtk_action_group_add_action (action_group,
						     action);

			G_GNUC_END_IGNORE_DEPRECATIONS
			if (surface != NULL) {
				g_object_set_data_full (G_OBJECT (action), "menu-icon",
							cairo_surface_reference (surface),
							(GDestroyNotify)cairo_surface_destroy);
			}

			g_object_set (action, "hide-if-empty", FALSE, NULL);

			g_object_unref (action);
		}
```
I am not really sure what consequences this can have.